### PR TITLE
issuee-167- new text on abou-us and added link to about_us in the he footer

### PR DESCRIPTION
**Pull request by Riscum**
image

image

### CODE REVIEW

File: app/javascript/components/about-us/AboutUsContributors.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default function AboutUsContributors() {
             <Row type="flex" justify="space-between" align="middle">
                 <Col span={24}>
                     <h2 className="about-us-contributors-title">Quem Somos</h2>
+                    <p>A comunidade Política Para Todos nasceu no verão de 2019 com o objetivo de promover a participação ativa dos cidadãos nos processos eleitorais em Portugal. É composta por voluntários de várias partes do país e diferentes áreas profissionais. Adotamos princípios de transparência radical na nossa forma de trabalho: toda a comunicação interna está disponível numa plataforma aberta , as reuniões são gravadas e também de acesso livre. A adesão à comunidade é aberta para qualquer cidadão.</p>
```
Comment (luisiul359) [inline app/javascript/components/about-us/AboutUsContributors.js]: I think text centered here does not resulta. I would suggest to add a new `Row` before this one with the paragraph.

File: app/javascript/components/common/LayoutFooter.js [hunk 2]
```diff
@@ -38,8 +38,7 @@ class LayoutFooter extends PureComponent {
                     </Col>
                     <Col span={24} md={{ offset: 4, span: 16 }} lg={{ offset: 0, span: 12 }} className="footer-info">
                         <h2>Quem Somos</h2>
-                        <p>Podes consultar <a target="_blank"
-                            rel="noopener" href="https://www.loomio.org/g/ZqT2uPv6/politica-para-todos">aqui</a> todo o histórico do projecto.</p>
+                        <p>Podes consultar <a href="/about-us">aqui</a> a equipa por detrás do projecto e as suas motivações.</p>
```
Comment (luisiul359) [inline app/javascript/components/common/LayoutFooter.js]: Add `rel="noopener"` for security reasons

**Comment by Riscum** [comment]
new iteration:
about us alignment 
image

footer update. link in "aqui" now follows to about_us
image

File: app/javascript/components/common/LayoutFooter.js (hunk 2 above)
Comment (Riscum) [reply app/javascript/components/common/LayoutFooter.js]: added on last commit,.

File: app/javascript/components/about-us/AboutUsContributors.js (hunk 1 above)
Comment (luisiul359) [inline app/javascript/components/about-us/AboutUsContributors.js]: `<br/>` after this paragraph. 
` aberta , ` remove space before comma

**Review by luisiul359 — APPROVED** [review]
LGTM

**Pull request merged**